include/alp/base/matrix.hpp

```diff
@@ -442,7 +442,8 @@ namespace alp {
 	} // namespace internal
 
 
-	template< typename T, typename Structure, enum Density density, typename View, enum Backend backend >
+	template< typename T, typename Structure, enum Density density, typename View,
+		typename ImfL, typename ImfR, enum Backend backend >
```

Comment:
Is ImfL something that affects rows and ImfR something that affects columns? (Maybe add documentation at least at the initial occurrence of these template arg names:)

Reply:
We adopted a new naming ImfR for rows, ImfC for columns. This accidentally remained, I will fix it.